packages/dashboard/src/lib/components/data-input/configurable-operation-list-input.tsx

```
@@ -0,0 +1,403 @@
+        switch (uiComponent) {
+            case 'boolean-form-input':
+                return (
+                    <Switch
+                        checked={itemValue === 'true'}
+                        onCheckedChange={checked => handleUpdateItem(index, checked.toString())}
+                        disabled={readOnly}
+                    />
+                );
+
+            case 'select-form-input':
+                const options = (definition.ui as any)?.options || [];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)